docs/install/tutorials/rpi.rst

```diff
@@ -375,10 +376,10 @@ Kolibri is under development with regards to optimizing performance on Raspberry
 
 This work will be released in a future package targeting a pre-configured UWSGI and Nginx implementation, which will be possible to add by replacing the package of this tutorial with the new package. We will add a release note and update this tutorial accordingly.
 
-You may also want to install other services such as `Kiwix <https://kiwix.org>`__. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to the one we added for Kolibri.
+You may also want to install other services such as `Kiwix <https://www.kiwix.org>`_. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to the one we added for Kolibri.
```

Comment:
switched from double- to single-underscore here. Intentional?

Reply:
Turns out Sphinx does not like duplicate explicit target references (TIL), si I had to turn them in anonymous:
https://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text/14067756#14067756